packages/server-wallet/server-wallet.api.md

```diff
@@ -223,15 +223,101 @@ export type SingleChannelOutput = {
     channelResult: ChannelResult;
 };
 
+// Warning: (ae-forgotten-export) The symbol "EventEmitterType" needs to be exported by the entry point index.d.ts
+// Warning: (ae-forgotten-export) The symbol "WalletInterface" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export class SingleThreadedWallet extends EventEmitter<EventEmitterType> implements WalletInterface {
```

Comment:
I intended for this file to show that the methods I made private disappear from this report. As it is, the file was not displaying this class at all until now, so we don't see the intended diff (but a much bigger one). 

The docs website is derived from this file, and we also enforce that it is kept up to date on github via a check on circle. (It is generated during `yarn prepare`, so will trigger `git diff --exit-code` unless it is checked in.